anta/tests/snmp.py

```diff
@@ -237,3 +240,121 @@ def test(self) -> None:
             self.result.is_failure(f"Expected `{self.inputs.contact}` as the contact, but found `{contact}` instead.")
         else:
             self.result.is_success()
+
+
+class VerifySNMPNotificationHost(AntaTest):
+    """Verifies the SNMP notification host (SNMP manager) configurations.
```

Comment:
can we have more brief description about what configuration we are verifying

Reply:
Updated docstring.